src/reducers/teamReducer.js

```diff
@@ -0,0 +1,27 @@
+import { LOAD_TEAMS_SUCCESS, TOGGLE_TEAM } from '../actions/actionTypes';
+import initialState from './initialState';
+
+const team = (state = {}, action) => {
+  switch(action.type) {
+    case TOGGLE_TEAM:
+      debugger
+      const regex = new RegExp(`(${action.team_name})`, "i");
```

Comment:
Regex is required - I use it for filtering teams by team_name. Someone types part of team_name and I have to check if it match or not.